portfolio/src/main/java/com/google/sps/servlets/Comments.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+public final class Comments {
```

Comment:
The name of this class suggests that it represents multiple comments, which is not accurate. Please rename to `Comment`